pkg/site/planning.go

```diff
@@ -0,0 +1,161 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package site
+
+import (
+	"fmt"
+	"html/template"
+	"net/http"
+	"path/filepath"
+	"strings"
+
+	"github.com/google/triage-party/pkg/hubbub"
+	"github.com/google/triage-party/pkg/provider"
+	"github.com/google/triage-party/pkg/triage"
+	"k8s.io/klog/v2"
+)
+
+const (
+	unplanned  = "Not Planned"
```

Comment:
Do you mind making the string value `Unplanned`? It's both shorter and more aligned with the variable name.